docs/docs/03-language-reference.md

```diff
@@ -1577,7 +1581,12 @@ of methods with different phases is not allowed as well.
 
 Interfaces represent a contract that a class must fulfill.
 Interfaces are defined with the `interface` keyword.
-Currently, preflight interfaces are allowed, while inflight interfaces are not supported yet (see https://github.com/winglang/wing/issues/1961).
+Interfaces may be either preflight interfaces or inflight interfaces.
+Preflight interfaces are defined in preflight scope and can contain both preflight and inflight methods.
+Only preflight classes may implement preflight interfaces.
+Inflight interfaces are either defined with the `inflight` modifier in preflight scope or simply defined in inflight scope.
+All methods of inflight interfaces are implicitly inflight (no need to use the `inflight` keyword).
+Both preflight and inflight classes may implement an inflight interface.
```

Comment:
The reason a preflight class can implement an inflight interface might not be obvious
```suggestion
Since both preflight and inflight classes can have inflight methods defined inside them, they are both capable of implementing inflight interfaces.
```